vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/TextBlock.java

```diff
@@ -46,9 +47,58 @@ public void setText(String text) {
     @Override
     @Transient
     public String htmlRenderedText() {
```

Comment:
the citations are in a biblio block, not a text block. So I don't think this is right.